euphony/src/main/java/co/euphony/tx/EuTxManager.java

```diff
@@ -132,21 +132,21 @@ public void stop()
 	}
 
 	/*
-	 * @deprecated Replaced by {@link #setCode()}, deprecated for naming & dynamic option.
+	 * @deprecated Replaced by {@link #setCode(String)}, deprecated for naming & dynamic option.
 	 */
 	@Deprecated
 	public void euInitTransmit(String data) {
 		setCode(data);
 	}
 
 	/*
-	 * @deprecated Replaced by {@link #setCode()}, deprecated for naming issue
+	 * @deprecated Replaced by {@link #play(int)}, deprecated for naming issue
```

Comment:
here should be play() not play(int).
I think it will be last one :)

after modify it, let me approve it!

Reply:
Thanks for missing part😂!! I applied feedback!